docs/content/en/docs/concepts/metrics/_index.md

```diff
@@ -0,0 +1,38 @@
+---
+title: Metrics
+description: Learn what Keptn Metrics are and how to use them
+icon: concepts
+layout: quickstart
+weight: 10
+hidechildren: true # this flag hides all sub-pages in the sidebar-multicard.html
+---
+
+### Keptn Metric
+A `KeptnMetric` is a CRD representing a metric. The metric will be collected from the provider specified in the specs.provider.name field. The query is a string in the provider-specific query language, used to obtain a metric. Providing the metrics as CRD into a K8s cluster will facilitate the reusability of this data across multiple components. Furthermore, this allows using multiple observability platforms for different metrics.
+
+A `KeptnMetric` looks like the following:
+
+```yaml
+apiVersion: metrics.keptn.sh/v1alpha1
+kind: KeptnMetric
+metadata:
+  name: keptnmetric-sample
+  namespace: keptn-lifecycle-toolkit-system
```

Comment:
should we also add a note about the requirement of metrics being applied only in the `keptn-lifecycle-toolkit-system` namespace here as well?